client/my-sites/plans/jetpack-plans/storage-tier-upgrade/style.scss

```
@@ -0,0 +1,18 @@
+.storage-tier-upgrade {
+	display: grid;
+	grid-template-columns: repeat( auto-fit, minmax( 300px, 1fr ) );
+	gap: 24px 16px;
+
+	margin-top: 48px;
+	margin-bottom: 64px;
+
```

Design mock-ups have the prices center-aligned here.

```suggestion

	.jetpack-product-card__price {
		text-align: center;
	}
	
```